### PR TITLE
Update namespace finder and service validation checker

### PR DESCRIPTION
**Pull request by dwlee**
To solve following issue
https://github.com/robotics-in-concert/rocon_tools/blob/indigo/rocon_python_comms/src/rocon_python_comms/services.py#L48

related PR
https://github.com/robotics-in-concert/rocon_qt_gui/pull/182

### CODE REVIEW

File: rocon_python_comms/src/rocon_python_comms/services.py [hunk 1]
```diff
@@ -36,6 +36,26 @@
 ##############################################################################
 
 
+def is_valid_service(service_name):
```
Comment (stonier) [inline rocon_python_comms/src/rocon_python_comms/services.py]: `is_valid_service` is confusing...that assumes it is available AND not bad/corrupt.

`is_available` or `service_is_available` would be less confusing.

Comment (jihoonl) [inline rocon_python_comms/src/rocon_python_comms/services.py]: `service_is_available`

**Comment by jihoonl** [comment]
:+1:

**Comment by jihoonl** [comment]
I am changing the logic of [find_namespace](https://github.com/robotics-in-concert/rocon_tools/blob/indigo/rocon_python_comms/src/rocon_python_comms/namespace.py#L36) by following. 
- renaming `find_namespace` to `find_service_namespace` - To be precise,  it finds namespace for ros service. We may develop `find_topic_namespace` later.
- provide `unique` argument just as [find_service](https://github.com/robotics-in-concert/rocon_tools/blob/indigo/rocon_python_comms/src/rocon_python_comms/services.py#L59) and returns multiple namespaces -  it is easier to handle code flow.
- `service_type` argument as optional - We may want to find namespace without type.

**Comment by jihoonl** [comment]
@stonier, @dwlee FYI

**Pull request merged**